### ISSUE TITLE
supporting other metrics for `elbow_curve`

### ISSUE THREAD

**Issue by edublancas** [ploomber/sklearn-evaluation#147]
Our current implementation of `elbow_curve` only supports plotting the sum of squares. However, there are other metrics that we can plot: average silhouette score (the line we draw on the silhouette analysis plot), [davies-boulding index](https://scikit-learn.org/stable/modules/clustering.html#davies-bouldin-index), and [Calinski-Harabasz Index](https://scikit-learn.org/stable/modules/clustering.html#calinski-harabasz-index). [see this for an example](https://www.scikit-yb.org/en/latest/api/cluster/elbow.html).

We should add a `metrics` argument to let the user choose which metric to plot.

**Comment by krutik2966** [comment]
Hello,

Is anyone already working on this issue? I would like to work on it.
